tests/kafkatest/services/kafka/kafka.py

```diff
@@ -893,8 +903,13 @@ def start_node(self, node, timeout_sec=60, **kwargs):
             # format log directories if necessary
             kafka_storage_script = self.path.script("kafka-storage.sh", node)
             cmd = "%s format --ignore-formatted --config %s --cluster-id %s" % (kafka_storage_script, KafkaService.CONFIG_FILE, config_property.CLUSTER_ID)
+            if self.kip853:
+                cmd += " --feature kraft.version=1"
+                if not self.first_controller_started and self.node_quorum_info.has_controller_role:
```

Comment:
why are we not specifying the `--initial-voters` here? seems wrong (for clusters with more than one controller)

Reply:
`--initial-controllers`? I think either way is correct based on [Jose's docs](https://github.com/apache/kafka/pull/17076/files#diff-e4e8d893dc2a4e999c96713dd5b5857203e0756860df0e70fb0cb041aa4d347bR3806). I would say not using it represents the point of our test a bit better - we initially wanted one controller but then later decide to add another one

Reply:
I think this is something we should add since I feel that `--initial-voters` is actually going to be more common. But let's not block the PR on this.